iModelCore/ECDb/ECDb/SchemaWriter.cpp

```diff
@@ -2713,7 +2720,10 @@ BentleyStatus SchemaWriter::UpdateCustomAttributes(Context& ctx, SchemaPersisten
                 {
                 // Not sure why the SchemaComparer found the new CA but it isn't here now; however, we can't fail the import because of this.
                 if (ctx.IgnoreIllegalDeletionsAndModifications())
+                    {
+                    ctx.Issues().ReportV(IssueSeverity::Info, IssueCategory::BusinessProperties, IssueType::ECDbIssue, ECDbIssueId::ECDb_0635, "Found a new custom attribute added that is null. Error suppressed as DoNotFailForDeletionsOrModifications flag is set.");

```

Comment:
Let's add the "ignoring upgrade error:" prefix to this and the following message, too. It makes them easier to put in context.

Reply:
Done !